app/src/main/java/com/safetyheads/akademiaandroida/login_placeholder.kt

```diff
@@ -0,0 +1,43 @@
+package com.safetyheads.akademiaandroida
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+import androidx.navigation.Navigation
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+class login_placeholder : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param1 = it.getString(ARG_PARAM1)
+            param2 = it.getString(ARG_PARAM2)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_login_placeholder, container, false)
+        val loginButton = view.findViewById<Button>(R.id.button_login)
+        loginButton.setOnClickListener {
+            Navigation.findNavController(view).navigate(R.id.dashboard_placeholder)
```

Comment:
R.id.action_login_placeholder_to_dashboard_placeholder should be used instead 🤔 
this applies to all ...navigate() calls you have

Reply:
use actions that are in XML instead of calling fragments directly